src/tests/test.json

```diff
@@ -0,0 +1,32 @@
+{
+  "zendesk_ticket_id": "0987654567",
+  "notify_message_id": "3456787654",
+    "event_type": "HOME_REPORT_SUSPICIOUS_ACTIVITY",
+  "user_id": "1234567",
+  "group_id": "466cac70-8923-4c25-b43b-631e8584dc3e", //check with ben what the name is
```

Comment:
Do we still need to find out what this ID is, or can we remove this comment from the test fixture?

Reply:
Addressed - The whole JSON file is removed, it was not needed as runtime code.